File: Layer_datas/Services/Objects/activities/Place.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Layer_datas.Services.Objects.activities
{
    public class Place
    {
        public int id { get; set; }
        public string name { get; set; }
        public string adress { get; set; }
        public string postalcode { get; set; }
        public string city { get; set; }

    }
}

File: Swiman/__scripts__/modules/api/models/Place.ts


    

    import * as assign from "object-assign";

    

    


    class Place {
        

        
        // ID
        public id: number = 0;
        // NAME
        public name: string = undefined;
        // ADRESS
        public adress: string = undefined;
        // POSTALCODE
        public postalcode: string = undefined;
        // CITY
        public city: string = undefined;

        constructor(init?: Partial<Place>){
            assign(this, init);
        }
        
    }


export {  Place }



